src/FirstSteps/RPG/HeroCantHoldItemException.cs

```diff
@@ -2,24 +2,24 @@
 
 namespace FirstSteps.RPG
 {
-    public class HandlingItemException : Exception
+    public class HeroCantHoldItemException : Exception
     {
         const string text = "You can't hold this item";
-        public HandlingItemException() : base(text)
+        public HeroCantHoldItemException() : base(text)
         {
             
         }
        public void TrySpendCoins()
```

Comment:
You don't need these extra methods. Take a look at the example from docs: [How to create user-defined exceptions](https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions)
```
using System;

public class EmployeeListNotFoundException : Exception
{
    public EmployeeListNotFoundException()
    {
    }

    public EmployeeListNotFoundException(string message)
        : base(message)
    {
    }

    public EmployeeListNotFoundException(string message, Exception inner)
        : base(message, inner)
    {
    }
}
```

You have here only constructors and that's basically it. So in your case, that one constructor you've created is enough.